src/MacVim/mvim

```diff
@@ -53,6 +53,11 @@ opts=
 # GUI mode, implies forking
 case "$name" in m*|g*|rm*|rg*) gui=true ;; esac
 
+# Logged in over SSH? No gui.
+if [ -n "${SSH_CONNECTION}" ]; then
+  gui=''
```

Comment:
>
> could you fix the indent? and gui= doesn't work (instead of gui='')
>
I've switched to tabs instead of spaces, and used an empty value. Please
take another look.

KT.